### PR TITLE
Refactor Code

### PR DESCRIPTION
**Pull request by mrkamin**
What does this PR do?

In this PR we Refactor My Code.

Description of the task to be completed 👎

In this project, we will use our newly acquired knowledge and refactor our previous code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,18 @@ To run tests, run the following command:
 - Twitter: [@twitterhandle](https://twitter.com/Mohamma63974237)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammad-rafi-amin-63b4319b/)
 
+👤 **Henschel Nketchogue M.**
```
Comment (Olamarx) [inline README.md:113]: - [x] You probably forgot the tech stack, kindly add it.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @mrkamin, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you have probably forgotten the tech stack, kindly add it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @TEAM     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] linters are passing
- [x] Professional README 💯
- [x] Refactored code based on requirement🥇
- [x] Console App working as intended 🥇  
 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

@KayLemba ,
thanks for your good and valuable feedback.

File: README.md (hunk 1 above)
Comment (mrkamin) [reply README.md:113]: @Olamarx,
thanks for your good and valuable feedback

**Pull request merged**